README.md

```diff
@@ -0,0 +1,14 @@
+# Lego Robot Web Service Project
+
+This repository contains the source code for our Java RESTful web service designed to control a Lego robot. This repository serves as the server side of the robot where the and also the client where the user can input new data for the robot to update itself.
+
+## Authors
+
+* Hongqian Li - Initial work - [HongqianLi](https://github.com/hongqian-li)
+* Richard Švajdlenka - Database Integration - [Richard](https://github.com/M3mberTM)
+* Julian Hack - System Testing - [JulianHack](https://github.com/Julian23000)
+* Attila Vegh - Backend Development - [AttilaVegh](https://github.com/Attila22000)
+
+
+### Hongqian Li
+Main responsibilities include ensuring the continuity and integration of content from the previous robotics project, facilitating team meetings, managing Jira and Confluence, and providing code and testing support to the team to complete basic initial work.
```

Comment:
This is very clearly written by ChatGPT. I understand you having hard time trying to write it in english but you should still try. It needs to be your words.

Reply:
Thanks for your feedback, I will do it by my own.